src/parse.rs

```diff
@@ -633,10 +633,23 @@ impl<'a> Parser<'a> {
             (Some(LookBehind), 3)
         } else if self.re[ix..].starts_with("?<!") {
             (Some(LookBehindNeg), 3)
-        } else if self.re[ix..].starts_with("?<") {
-            // Named capture group using Oniguruma syntax: (?<name>...)
+        } else if self.re[ix..].starts_with("?<") || self.re[ix..].starts_with("?'") {
+            // Named capture group using Oniguruma syntax: (?<name>...) or (?'name'...)
             self.curr_group += 1;
-            if let Some((id, skip)) = parse_id(&self.re[ix + 1..], "<", ">", false) {
+            if let Some((id, skip)) = parse_id(
+                &self.re[ix + 1..],
+                if self.re[ix..].starts_with("?<") {
+                    "<"
+                } else {
+                    "'"
+                },
+                if self.re[ix..].starts_with("?<") {
+                    ">"
+                } else {
+                    "'"
+                },
```

Comment:
Let's pull these out as variables, something like?:

```suggestion
let (open, close) = if self.re[ix..].starts_with("?<") {
                    ("<", ">")
                } else {
                    ("'", "'")
                };
```

Reply:
great idea, thanks - done :slightly_smiling_face: